timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/LockCollection.java

```diff
@@ -15,41 +15,13 @@
  */
 package com.palantir.atlasdb.timelock.lock;
 
-import com.github.benmanes.caffeine.cache.Caffeine;
-import com.github.benmanes.caffeine.cache.LoadingCache;
-import com.google.common.collect.Lists;
 import com.palantir.lock.LockDescriptor;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.List;
 import java.util.Set;
 
-public class LockCollection {
+final class LockCollection {
```

Comment:
This is one of those sufficiently old things where you'll need to check if anyone is doing some reflection into here using our internal tooling.

Reply:
Good flag, sent some evidence internally